build/common.project.props

```diff
@@ -178,8 +178,8 @@
       <_Parameter1>$(SemanticVersion).$(PreReleaseVersion)</_Parameter1>
     </AssemblyAttributes>
     <AssemblyAttributes Include="AssemblyInformationalVersion">
-      <_Parameter1 Condition="'$(BUILD_SOURCEVERSION)' == ''">$(SemanticVersion)$(PreReleaseInformationVersion)</_Parameter1>
-      <_Parameter1 Condition="'$(BUILD_SOURCEVERSION)' != ''">$(SemanticVersion)$(PreReleaseInformationVersion)+$(BUILD_SOURCEVERSION)</_Parameter1>
+      <_Parameter1>$(SemanticVersion)$(PreReleaseInformationVersion)</_Parameter1>
```

Comment:
why change this? 
This wouldn't impact semver.

Reply:
trying different things, nothing has worked so far. I still cant figure out where it's adding that commitsha with package version.

If you know where it does, let me know!